plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/ptf/Query.java

```diff
@@ -62,26 +62,29 @@
     public static final String NAME = "query";
 
     private final MongoMetadata metadata;
```

Comment:
@ebyhr 

If we create a `MongoMetadata` object in the constructor of QueryFunction, will it not create an unnecessary object because a MongoMetadata object is also created whenever a QueryFunction is created? I'm curious about your opinion. If the cost is not high, I will move the `MongoMetadata` object creation statement inside the QueryFunction.